Tooltip: Add screener tests and fix high contrast border

behowell opened the pull request:
#### Pull request checklist

- [x] Addresses an existing issue: 
  - Addresses #20934
  - Fixes #20177 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

* Add screener stories for Tooltip
* Add a border to Tooltip using `colorTransparentStroke`, which is transparent in all themes except high contrast
* Fix `createArrowStyles`:
  * Use `width` _and_ `height` to set the arrow's size instead of  `aspectRatio`, which wasn't working as expected, and was producing a zero-size tooltip.
  * Accept border styling parameters, and apply them to the arrow's border.

Screenshots of before and after the styling changes:
![image](https://user-images.githubusercontent.com/48106640/148048625-cd7aec74-df5e-4728-9536-ae1e61743691.png)

fabricteam commented:
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                                                      Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `169.082 kB`<br />`47.656 kB` | `168.853 kB`<br />`47.593 kB` | `-229 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /><br />`-63 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /> |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `95.761 kB`<br />`29.129 kB` |  `95.532 kB`<br />`29.064 kB` | `-229 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /><br />`-65 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /> |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `44.566 kB`<br />`15.426 kB` |  `44.315 kB`<br />`15.351 kB` | `-251 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /><br />`-75 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" /> |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                  |          Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> |   `32.184 kB`<br />`10.58 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                               | `103.621 kB`<br />`31.866 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                  | `105.878 kB`<br />`32.212 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePopper</abbr>                                              |   `22.808 kB`<br />`7.935 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/cc44f3625c6c6b5775bc08f00c5b8f52145ad52b'>cc44f3625c6c6b5775bc08f00c5b8f52145ad52b</a></sub>
<div id="bundleSizeComment9423"/>

fabricteam commented:
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Avatar-mount_base.html">945</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Avatar-mount.html">888</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/BaseButton-mount_base.html">985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/BaseButton-mount.html">996</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Breadcrumb-mount_base.html">2813</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Breadcrumb-mount.html">2811</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ButtonNext-mount_base.html">554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ButtonNext-mount.html">562</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Checkbox-mount_base.html">1586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Checkbox-mount.html">1578</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/CheckboxBase-mount_base.html">1385</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/CheckboxBase-mount.html">1340</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ChoiceGroup-mount_base.html">5018</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ChoiceGroup-mount.html">4971</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ComboBox-mount_base.html">1062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ComboBox-mount.html">1014</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/CommandBar-mount_base.html">10602</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/CommandBar-mount.html">10461</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ContextualMenu-mount_base.html">8878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ContextualMenu-mount.html">8789</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DefaultButton-mount_base.html">1212</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DefaultButton-mount.html">1233</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DetailsRow-mount_base.html">4002</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DetailsRow-mount.html">4015</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DetailsRowFast-mount_base.html">3943</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DetailsRowFast-mount.html">3956</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DetailsRowNoStyles-mount_base.html">3788</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DetailsRowNoStyles-mount.html">3787</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Dialog-mount_base.html">2689</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Dialog-mount.html">2733</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DocumentCardTitle-mount_base.html">193</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/DocumentCardTitle-mount.html">199</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Dropdown-mount_base.html">3374</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Dropdown-mount.html">3404</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FluentProviderNext-mount_base.html">1962</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FluentProviderNext-mount.html">1920</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FluentProviderWithTheme-mount_base.html">169</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FluentProviderWithTheme-mount.html">181</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FluentProviderWithTheme-virtual-rerender.html">134</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">211</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">192</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FocusTrapZone-mount_base.html">1886</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FocusTrapZone-mount.html">1991</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FocusZone-mount_base.html">1901</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/FocusZone-mount.html">2049</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/IconButton-mount_base.html">1831</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/IconButton-mount.html">1822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Label-mount_base.html">403</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Label-mount.html">418</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Layer-mount_base.html">3070</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Layer-mount.html">3144</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Link-mount_base.html">513</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Link-mount.html">541</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/MakeStyles-mount_base.html">1962</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/MakeStyles-mount.html">1917</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/MenuButton-mount_base.html">1551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/MenuButton-mount.html">1530</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/MessageBar-mount_base.html">2142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/MessageBar-mount.html">2103</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Nav-mount_base.html">3389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Nav-mount.html">3472</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/OverflowSet-mount_base.html">1226</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/OverflowSet-mount.html">1205</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Panel-mount_base.html">2571</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Panel-mount.html">2536</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Persona-mount_base.html">884</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Persona-mount.html">886</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Pivot-mount_base.html">1545</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Pivot-mount.html">1513</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/PrimaryButton-mount_base.html">1359</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/PrimaryButton-mount.html">1368</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Rating-mount_base.html">7960</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Rating-mount.html">7972</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/SearchBox-mount_base.html">1398</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/SearchBox-mount.html">1383</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Shimmer-mount_base.html">2627</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Shimmer-mount.html">2581</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Slider-mount_base.html">2059</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Slider-mount.html">2100</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/SpinButton-mount_base.html">5228</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/SpinButton-mount.html">5212</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Spinner-mount_base.html">486</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Spinner-mount.html">461</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/SplitButton-mount_base.html">3450</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/SplitButton-mount.html">3284</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Stack-mount_base.html">539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Stack-mount.html">571</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/StackWithIntrinsicChildren-mount_base.html">1803</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/StackWithIntrinsicChildren-mount.html">1760</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/StackWithTextChildren-mount_base.html">4839</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/StackWithTextChildren-mount.html">4732</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/SwatchColorPicker-mount_base.html">10941</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/SwatchColorPicker-mount.html">10890</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/TagPicker-mount_base.html">2753</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/TagPicker-mount.html">2721</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/TeachingBubble-mount_base.html">13263</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/TeachingBubble-mount.html">13412</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Text-mount_base.html">453</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Text-mount.html">458</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/TextField-mount_base.html">1431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/TextField-mount.html">1434</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ThemeProvider-mount_base.html">1276</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ThemeProvider-mount.html">1285</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ThemeProvider-virtual-rerender_base.html">665</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ThemeProvider-virtual-rerender.html">637</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1976</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1970</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Toggle-mount_base.html">881</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/Toggle-mount.html">886</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/buttonNative-mount_base.html">163</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test/buttonNative-mount.html">154</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
PortalMinimalPerf.default | [207](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/PortalMinimalPerf.default.html) | [187](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.11:1
ReactionMinimalPerf.default | [425](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ReactionMinimalPerf.default.html) | [386](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.1:1
HeaderMinimalPerf.default | [414](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/HeaderMinimalPerf.default.html) | [379](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.09:1
TextAreaMinimalPerf.default | [538](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TextAreaMinimalPerf.default.html) | [492](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.09:1
LayoutMinimalPerf.default | [390](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/LayoutMinimalPerf.default.html) | [366](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.07:1
StatusMinimalPerf.default | [781](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/StatusMinimalPerf.default.html) | [732](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.07:1
ButtonMinimalPerf.default | [186](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ButtonMinimalPerf.default.html) | [175](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.06:1
GridMinimalPerf.default | [358](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/GridMinimalPerf.default.html) | [338](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.06:1
ProviderMergeThemesPerf.default | [1837](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1740](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1.06:1
AnimationMinimalPerf.default | [478](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AnimationMinimalPerf.default.html) | [456](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.05:1
AttachmentSlotsPerf.default | [1153](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AttachmentSlotsPerf.default.html) | [1097](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.05:1
AvatarMinimalPerf.default | [213](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AvatarMinimalPerf.default.html) | [202](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.05:1
CarouselMinimalPerf.default | [506](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/CarouselMinimalPerf.default.html) | [480](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.05:1
ItemLayoutMinimalPerf.default | [1253](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [1198](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.05:1
SegmentMinimalPerf.default | [370](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/SegmentMinimalPerf.default.html) | [351](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.05:1
AlertMinimalPerf.default | [291](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AlertMinimalPerf.default.html) | [279](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1.04:1
DropdownManyItemsPerf.default | [729](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DropdownManyItemsPerf.default.html) | [701](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.04:1
RadioGroupMinimalPerf.default | [479](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [460](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.04:1
ChatMinimalPerf.default | [786](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ChatMinimalPerf.default.html) | [761](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.03:1
MenuButtonMinimalPerf.default | [1755](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1704](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.03:1
CardMinimalPerf.default | [598](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/CardMinimalPerf.default.html) | [585](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.02:1
DialogMinimalPerf.default | [825](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DialogMinimalPerf.default.html) | [809](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.02:1
FlexMinimalPerf.default | [314](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/FlexMinimalPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.02:1
ListNestedPerf.default | [615](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ListNestedPerf.default.html) | [604](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ListNestedPerf.default_base.html) | 1.02:1
LoaderMinimalPerf.default | [726](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/LoaderMinimalPerf.default.html) | [713](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.02:1
SkeletonMinimalPerf.default | [376](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/SkeletonMinimalPerf.default.html) | [370](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.02:1
TableMinimalPerf.default | [425](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TableMinimalPerf.default.html) | [417](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.02:1
TreeMinimalPerf.default | [820](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TreeMinimalPerf.default.html) | [804](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.02:1
DividerMinimalPerf.default | [384](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DividerMinimalPerf.default.html) | [381](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.01:1
InputMinimalPerf.default | [1378](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/InputMinimalPerf.default.html) | [1358](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.01:1
ListWith60ListItems.default | [669](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ListWith60ListItems.default.html) | [664](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ListWith60ListItems.default_base.html) | 1.01:1
SliderMinimalPerf.default | [1770](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/SliderMinimalPerf.default.html) | [1753](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.01:1
SplitButtonMinimalPerf.default | [4397](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [4342](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.01:1
TableManyItemsPerf.default | [1982](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TableManyItemsPerf.default.html) | [1953](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.01:1
DatepickerMinimalPerf.default | [5883](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DatepickerMinimalPerf.default.html) | [5912](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1:1
ImageMinimalPerf.default | [406](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ImageMinimalPerf.default.html) | [405](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1:1
ListMinimalPerf.default | [542](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ListMinimalPerf.default.html) | [540](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ListMinimalPerf.default_base.html) | 1:1
ProviderMinimalPerf.default | [1166](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ProviderMinimalPerf.default.html) | [1164](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1:1
TextMinimalPerf.default | [366](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TextMinimalPerf.default.html) | [365](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TextMinimalPerf.default_base.html) | 1:1
VideoMinimalPerf.default | [660](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/VideoMinimalPerf.default.html) | [659](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1:1
AttachmentMinimalPerf.default | [168](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AttachmentMinimalPerf.default.html) | [170](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 0.99:1
CheckboxMinimalPerf.default | [2748](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2778](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 0.99:1
CustomToolbarPrototype.default | [4253](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/CustomToolbarPrototype.default.html) | [4281](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 0.99:1
ButtonOverridesMissPerf.default | [1726](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1770](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 0.98:1
ButtonSlotsPerf.default | [558](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ButtonSlotsPerf.default.html) | [572](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 0.98:1
ChatWithPopoverPerf.default | [406](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ChatWithPopoverPerf.default.html) | [414](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.98:1
DropdownMinimalPerf.default | [3049](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DropdownMinimalPerf.default.html) | [3124](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 0.98:1
EmbedMinimalPerf.default | [4227](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/EmbedMinimalPerf.default.html) | [4321](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 0.98:1
ListCommonPerf.default | [650](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ListCommonPerf.default.html) | [665](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ListCommonPerf.default_base.html) | 0.98:1
IconMinimalPerf.default | [661](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/IconMinimalPerf.default.html) | [673](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/IconMinimalPerf.default_base.html) | 0.98:1
BoxMinimalPerf.default | [355](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/BoxMinimalPerf.default.html) | [367](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/BoxMinimalPerf.default_base.html) | 0.97:1
FormMinimalPerf.default | [422](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/FormMinimalPerf.default.html) | [435](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/FormMinimalPerf.default_base.html) | 0.97:1
LabelMinimalPerf.default | [391](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/LabelMinimalPerf.default.html) | [402](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/LabelMinimalPerf.default_base.html) | 0.97:1
MenuMinimalPerf.default | [868](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/MenuMinimalPerf.default.html) | [892](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/MenuMinimalPerf.default_base.html) | 0.97:1
ToolbarMinimalPerf.default | [954](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ToolbarMinimalPerf.default.html) | [984](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.97:1
TooltipMinimalPerf.default | [1085](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TooltipMinimalPerf.default.html) | [1118](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 0.97:1
TreeWith60ListItems.default | [186](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TreeWith60ListItems.default.html) | [191](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/TreeWith60ListItems.default_base.html) | 0.97:1
AccordionMinimalPerf.default | [166](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AccordionMinimalPerf.default.html) | [173](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.96:1
ChatDuplicateMessagesPerf.default | [316](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [334](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 0.95:1
RosterPerf.default | [1161](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/RosterPerf.default.html) | [1227](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/RosterPerf.default_base.html) | 0.95:1
HeaderSlotsPerf.default | [758](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/HeaderSlotsPerf.default.html) | [810](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 0.94:1
PopupMinimalPerf.default | [612](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/PopupMinimalPerf.default.html) | [649](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/PopupMinimalPerf.default_base.html) | 0.94:1
RefMinimalPerf.default | [235](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/RefMinimalPerf.default.html) | [251](https://fluentuipr.z22.web.core.windows.net/pull/21086/perf-test-northstar/RefMinimalPerf.default_base.html) | 0.94:1

</details><div id="perfComment9423"/>

layershifter commented on change/@fluentui-react-tooltip-9e0085eb-2ad8-4d35-9721-29b6f30fc9fe.json:
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Remove `inverted` prop that was added back by a merge conflict",
```
Can you please extract this change into a separate PR?

- it's not related to changes that this PR proposes
- it creates an entry in a changelog that does not match changes in the PR

layershifter commented on packages/react-positioning/src/createArrowStyles.ts:
```diff
@@ -1,80 +1,102 @@
+  /**
+   * The border of the arrow. This should be the same border as the parent element.
+   */
+  border?: [/*width:*/ string, /*style:*/ string, /*color:*/ string];
```
Can we use separate object properties there, i.e. `borderWidth`, `borderStyle` & `borderColor` separately?

```suggestion
  borderColor?: MakeStylesStyle['borderColor'],
  borderStyle?: MakeStylesStyle['borderStyle'],
  borderWidth?: MakeStylesStyle['borderWidth']
```

(👆 _didn't test it, but should work_)

- it will be more readable
- removes need to have comments to describe the tuple
- is not order dependent
- as styles are compiled anyway - does not have any effect on bundle size

layershifter commented:
> * Fix arrow styling in `createArrowStyles`

Can you please update PR's description to include before/after (normal & HC themes)? From the code it's not clear what is fixed.

behowell commented:
> Can you please update PR's description to include before/after (normal & HC themes)? From the code it's not clear what is fixed.

I've updated the description and added some screenshots.

layershifter commented:
I also checked for `Popover`, it's also broken on `master` 🙃

![image](https://user-images.githubusercontent.com/14183168/148069003-2ef95679-1d92-4696-8d5a-c3e0ccf5fecf.png)

behowell commented:
Yeah, this should fix popover too

khmakoto commented on packages/react-tooltip/src/components/Tooltip/useTooltip.tsx:
```diff
@@ -15,7 +15,7 @@ import type { TooltipProps, TooltipState, TooltipTriggerProps } from './Tooltip.
 
 // Style values that are required for popper to properly position the tooltip
 const tooltipBorderRadius = 4; // Update the root's borderRadius in useTooltipStyles.ts if this changes
-const arrowHeight = 6; // Update the arrow's width/height in useTooltipStyles.ts if this changes
+const arrowHeight = 6; // Update arrowHeight in useTooltipStyles.ts if this changes
```
Not blocking for this PR: Is this a case for maybe exporting out of one file and importing into another while not exporting from the package itself so we don't have duplicate information?

behowell replied:
I'll make that change in a follow-up PR

behowell replied:
#21204
